landscapes/ticuna/landscape.ini

```diff
@@ -0,0 +1,28 @@
+[landscape]
+
+name = Ticuna
+author = Priscila Faulhaber images combined into this 360-degree view.
+
+type = spherical
+
+maptex = ticuna1.png
```

Comment:
Some file systems are case sensitive (e.g. by default on Linux and MacOS) and char "T" and "t" is different chars in this case and it will case serious error. Please update maptex parameter or rename file.

Reply:
I had already made this change when I tested on Ubuntu. I already updated the files.